[Tesla] Fix data parsing after latest firmware upgrade

There a a lot of these

2017-06-27 09:10:37.665 [ERROR] [b.binding.tesla.handler.TeslaHandler] - An exception occurred while parsing data received from the vehicle: 'java.lang.IllegalStateException: Expected an int but was BOOLEAN at line 1 column 212 path $.is_front_defroster_on'

after the latest firmware (17.24.28) that was pushed by Tesla.

--- a/addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/protocol/ClimateState.java
+++ b/addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/protocol/ClimateState.java
@@ -21,7 +21,7 @@ public class ClimateState {
     public float driver_temp_setting;
     public float passenger_temp_setting;
     public boolean is_auto_conditioning_on;
-    public int is_front_defroster_on;
+    public boolean is_front_defroster_on;
     public boolean is_rear_defroster_on;
     public int fan_status;
     public int seat_heater_left;

